pkg/iostreams/regex_filter_writer.go

```diff
@@ -0,0 +1,46 @@
+package iostreams
```

Comment:
I was unsure what package to put this in. It seemed like an IO utility so I put it here, but am open to other suggestions.

Reply:
I don't think this writer needs to be generalized! Since we only use it for solving a `pr create` problem, it can live in the `pr/create` package.